src/server/routes/student.route.js

```diff
@@ -0,0 +1,7 @@
+import express from 'express';
+import { createStudentController } from '../../controller/students/createStudentController';
+const router = express.Router();
+
+router.post('/createStudent',  createStudentController);
```

Comment:
the same. If you use the verb `post` it is implicit that you are creating an student.

Reply:
Updated in https://github.com/Sly-Agustin/nosql-practice/commit/32e06334cb14ef863c9d06941c53a4b8bee84c7b